README.md

```diff
@@ -12,22 +12,24 @@ This repository contains the reference implementation for an Mission-Critical "o
 
 ## Reference implementation - Table of Contents
 
-- [Reference Implementation Solution Guide](./docs/reference-implementation/README.md) - Everything required to understand and build a copy of the reference implementation
-- [Reference Implementation Build Artifacts](./src/infra/README.md) - Contains the Infrastructure-as-Code artifacts, CI/CD pipelines, and application code required to deploy the pre-configured reference solution
+- [Getting Started](./docs/reference-implementation/Getting-Started.md) outlines the process and required steps to deploy Azure Mission-Critical in your environment, including preparing the Azure DevOps pipelines.
+- [Reference Implementation Solution Guide](./docs/reference-implementation/README.md) contains everything required to understand and build a copy of the reference implementation.
+- [Reference Implementation Build Artifacts](./src/infra/README.md) contains the Infrastructure-as-Code artifacts, CI/CD pipelines, and application code required to deploy the reference implementation.
```

Comment:
Add a link to the Reference architecture for this implementation: https://docs.microsoft.com/azure/architecture/reference-architectures/containers/aks-mission-critical/mission-critical-intro

Reply:
Re-ordered the landing page and added a link to the baseline architecture. 

<img width="593" alt="image" src="https://user-images.githubusercontent.com/17407022/185953634-34969f17-143a-40e6-8da3-2c0f27f1d271.png">